src/test/java/com/at/globalclasses/BasicSecurityUtil.java

```diff
@@ -6,7 +6,7 @@
     public String environment, requestBody, responseBody, apiResource;
     public ResponseEntity<String> response;
     public ApiTools ServiceApi;
-    public String type;
+    public String type, uridb;
```

Comment:
Why did you add this new variable to this class?

Reply:
Because I use these variable to call the DB URL, and the firts version of this code use a similar variable to call the path to make the GET request.
Do I have to change it?